rails-tasks/app/controllers/tasks_controller.rb

```diff
@@ -1,2 +1,27 @@
+require "pry"
+
 class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+    @task_name = params[:name]
+    task = Task.where(name: @task_name)[0]
+    @description = task.description
```

Comment:
Since these things are coming from the task itself, I think it would make more sense to utilize `@task.description` and `@task.complete` instead of creating separate instance variables for each property